pkg/document/document.go

```diff
@@ -116,6 +116,10 @@ func (d *Document) splitAndParse() error {
 	return nil
 }
 
+func (d *Document) FrontMatterRaw() []byte {
```

Comment:
Suggestion: Let's stick with `Frontmatter` for consistency.

```suggestion
func (d *Document) FrontmatterRaw() []byte {
```